src/Hooks/RequestLifecycleIsLongerThanHandler.php

```diff
@@ -44,9 +46,11 @@ public function __invoke(Carbon $startedAt, Request $request, Response $response
             $this->nightwatch->report($e);
         }
 
-        $this->nightwatch->ingest();
+        $this->nightwatch->digest();
 
         // TODO: Move this to an Octane-only hook.
-        $this->nightwatch->state->reset();
+        $this->nightwatch->state->flush();
+        // $this->nightwatch->ingest->flush();
+        // memory_reset_peak_usage();
```

Comment:
Adding these here for consistency and so we don't forget when adding Octane support